bridge/client/app/_components/ktb-project-settings-git-extended/ktb-project-settings-git-extended.component.ts

```diff
@@ -70,9 +94,19 @@ export class KtbProjectSettingsGitExtendedComponent {
       });
   }
 
+  private isRemoteUrlEmpty(): boolean {
+    if (!this.gitInputData) {
+      return true;
+    }
+    return (
+      (isGitHTTPS(this.gitInputData) && !this.gitInputData.https.gitRemoteURL) ||
+      (isGitSSH(this.gitInputData) && !this.gitInputData.ssh.gitRemoteURL)
+    );
+  }
+
   public setSelectedForm($event: DtRadioChange<GitFormType>): void {
     this.selectedForm = $event.value ?? GitFormType.HTTPS;
-    this.dataChanged(this.gitData);
+    this.dataChanged($event.value ?? GitFormType.HTTPS, this.gitData);
```

Comment:
duplicated assignment
`this.dataChanged(this.selectedForm, this.gitData)` to remove redundancy here.